Cabal/doc/cabal-commands.rst

```diff
@@ -99,6 +105,12 @@ Some example targets:
 
     $ cabal v2-build lib:foo-pkg       # build the library named foo-pkg
     $ cabal v2-build foo-pkg:foo-tests # build foo-tests in foo-pkg
+    $ cabal v2-build src/Lib.s         # build the library component to
+                                       # which "src/Lib.hs" belongs
+    $ cabal v2-build app/Main.hs       # build the executable component of
+                                       # "app/Main.hs"
+    $ cabal v2-build Lib               # build the library component to
+                                       # which the module "Lib" belongs
 
 (There is also syntax for specifying module and file targets, but it
 doesn't currently do anything.)
```

Comment:
Isn't this no longer true now?

Reply:
Why not? It was at least a month ago.

Reply:
Nothing here that this PR would have changed, did I miss another change?

Reply:
Ok, got it what you mean, thanks for catching this!